packages/react-core/src/components/EmptyState/examples/EmptyStateCustomIconColor.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateHeader,
+  EmptyStateIcon
+} from '@patternfly/react-core';
+import CubesIcon from '@patternfly/react-icons/dist/esm/icons/cubes-icon';
+
+export const EmptyStateCustomIconColor: React.FunctionComponent = () => (
+  <EmptyState>
+    <EmptyStateHeader
+      titleText="Custom icon color"
+      headingLevel="h4"
+      icon={<EmptyStateIcon icon={CubesIcon} color="#3e8635" />}
```

Comment:
@thatblindgeye are there values or types of values that we'd recommend using to set as the value of a CSS color variable rather than hardcoded hex values? Maybe this is exactly how it would be intended to be used, but usually hex values raise red flags for me.

Reply:
You are right, it is probably not a good idea, I have changed it to `color="var(--pf-global--success-color--100)"` in a new commit